### PR TITLE
docs(account-lib): add light copyedit

### PR DESCRIPTION
**Pull request by gbarr01**
add light copyedit

JIRA: BG-36902

### CODE REVIEW

File: modules/account-lib/DEVELOPER.md [hunk 1]
```diff
@@ -39,28 +45,49 @@ const signedTxJson = tx.toJson();
...
+for Tron). This is where the meat of the signing, validation, and encoding logic
+lives for each blockchain supported by the library.
 
-It is recommended to follow this pattern when adding a new coin to the library.
+We recommend that you follow this pattern when adding a new coin to the library:
 
 ### Test Structure
-This library comes with unit tests and it is expected that all changes introduced to the library increase test coverage. Tests live in `test`. Coin specific tests live in `test/unit/coin/<coin-ticker>/<coin-ticker.js>`.
 
+Accoutn Lib comes with unit tests and it is expected that all changes introduced
```
Comment (gbarr01) [inline modules/account-lib/DEVELOPER.md:63]: typo

**Review by tylerlevine — CHANGES_REQUESTED** [review]
This repo doesn't support PRs from forks, which is why you are getting the test failures. Please re-create this PR by pushing your branch to this repo and creating the PR from that branch.

**Comment by gbarr01** [comment]
> This repo doesn't support PRs from forks, which is why you are getting the test failures. Please re-create this PR by pushing your branch to this repo and creating the PR from that branch.

Thanks, @tylerlevine , I've tried pushing directly but cannot. Do you have any ideas? 

image

**Comment by tylerlevine** [comment]
> > This repo doesn't support PRs from forks, which is why you are getting the test failures. Please re-create this PR by pushing your branch to this repo and creating the PR from that branch.
> 
> Thanks, @tylerlevine , I've tried pushing directly but cannot. Do you have any ideas?
> 
> image

please open a ticket with the infra team to get added as a contributor to the repo.

**Pull request closed without merging**